include/libopencm3/stm32/h7/rcc.h

```diff
@@ -0,0 +1,458 @@
+  RST_GPIOB       = _REG_BIT(0x88, 1),
+  RST_GPIOC       = _REG_BIT(0x88, 2),
+  RST_GPIOD       = _REG_BIT(0x88, 3),
+  RST_GPIOE       = _REG_BIT(0x88, 4),
+  RST_GPIOF       = _REG_BIT(0x88, 5),
+  RST_GPIOG       = _REG_BIT(0x88, 6),
+  RST_GPIOH       = _REG_BIT(0x88, 7),
+  RST_GPIOI       = _REG_BIT(0x88, 8),
+  RST_GPIOJ       = _REG_BIT(0x88, 9),
+  RST_GPIOK       = _REG_BIT(0x88, 10),
+  RST_CRC         = _REG_BIT(0x88, 19),
+  RST_BDMA        = _REG_BIT(0x88, 21),
+  RST_ADC3        = _REG_BIT(0x88, 24),
+  RST_HSEM        = _REG_BIT(0x88, 25),
+
+  /* APB1L peripherals*/
+  RST_TIM2        = _REG_BIT(0x90, 0),
+  RST_TIM3        = _REG_BIT(0x90, 1),
+  RST_TIM4        = _REG_BIT(0x90, 2),
+  RST_TIM5        = _REG_BIT(0x90, 3),
+  RST_TIM6        = _REG_BIT(0x90, 4),
+  RST_TIM7        = _REG_BIT(0x90, 5),
+  RST_TIM12       = _REG_BIT(0x90, 6),
+  RST_TIM13       = _REG_BIT(0x90, 7),
+  RST_TIM14       = _REG_BIT(0x90, 8),
+  RST_LPTIM1      = _REG_BIT(0x90, 9),
+  RST_SPI2        = _REG_BIT(0x90, 14),
+  RST_SPI3        = _REG_BIT(0x90, 15),
+  RST_SPDIFRX     = _REG_BIT(0x90, 16),
+  RST_USART2      = _REG_BIT(0x90, 17),
+  RST_USART3      = _REG_BIT(0x90, 18),
+  RST_UART4       = _REG_BIT(0x90, 19),
+  RST_UART5       = _REG_BIT(0x90, 20),
+  RST_I2C1        = _REG_BIT(0x90, 21),
+  RST_I2C2        = _REG_BIT(0x90, 22),
+  RST_I2C3        = _REG_BIT(0x90, 23),
+  RST_CEC         = _REG_BIT(0x90, 27),
+  RST_DAC         = _REG_BIT(0x90, 29),
+  RST_UART7       = _REG_BIT(0x90, 30),
+  RST_UART8       = _REG_BIT(0x90, 31),
+
+  /* APB1H peripherals*/
+  RST_CRS         = _REG_BIT(0x94, 1),
+  RST_SWP         = _REG_BIT(0x94, 2),
+  RST_OPAMP       = _REG_BIT(0x94, 4),
+  RST_MDIO        = _REG_BIT(0x94, 5),
+  RST_FDCAN       = _REG_BIT(0x94, 8),
+
+  /* APB2 peripherals */
+  RST_TIM1        = _REG_BIT(0x98, 0),
+  RST_TIM8        = _REG_BIT(0x98, 1),
+  RST_USART1      = _REG_BIT(0x98, 4),
+  RST_USART6      = _REG_BIT(0x98, 5),
+  RST_SPI1        = _REG_BIT(0x98, 12),
+  RST_SPI4        = _REG_BIT(0x98, 13),
+  RST_TIM15       = _REG_BIT(0x98, 16),
+  RST_TIM16       = _REG_BIT(0x98, 17),
+  RST_TIM17       = _REG_BIT(0x98, 18),
+  RST_SPI5        = _REG_BIT(0x98, 20),
+  RST_SAI1        = _REG_BIT(0x98, 22),
+  RST_SAI2        = _REG_BIT(0x98, 23),
+  RST_SAI3        = _REG_BIT(0x98, 24),
+  RST_DFSDM       = _REG_BIT(0x98, 28),
+  RST_HRTIM       = _REG_BIT(0x98, 29),
+
+  /* APB3 peripherals */
+  RST_LTDCRST     = _REG_BIT(0x8C, 3),
+
+  /* APB4 peripherals */
+  RST_SYSCFG      = _REG_BIT(0x9C, 1),
+  RST_LPUART1     = _REG_BIT(0x9C, 3),
+  RST_SPI6        = _REG_BIT(0x9C, 5),
+  RST_I2C4        = _REG_BIT(0x9C, 7),
+  RST_LPTIM2      = _REG_BIT(0x9C, 9),
+  RST_LPTIM3      = _REG_BIT(0x9C, 10),
+  RST_LPTIM4      = _REG_BIT(0x9C, 11),
+  RST_LPTIM5      = _REG_BIT(0x9C, 12),
+  RST_COMP12      = _REG_BIT(0x9C, 14),
+  RST_VREF        = _REG_BIT(0x9C, 15),
+  RST_SAI4        = _REG_BIT(0x9C, 21),
+};
```

Comment:
You're missing all the SCC_ bits for the xxxLPENR registers.

Reply:
That's true, but I figured they could be added later if needed. I'm not sure how many H7 users are going to be looking for LP operation, honestly.

Reply:
just for consistency :)